pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py

```diff
@@ -57,7 +57,7 @@ def process(self, context):
                         instance.data["families"]
                     )
                 )[0]
-                instance.data["outputPath"] = os.path.join(
+                instance.data["output"] = os.path.join(
```

Comment:
Cool, I can go back to the original variable name.

```mixedCase``` has been suggested by Marcus for any data members, which I have been stuck to.